### PR TITLE
[BEAM-4333] Add integration tests for python mobile games

### PR DESCRIPTION
**Pull request by mariapython**
- [x] Add IT test for examples 1-3 (user score, hourly team score, leader board).
 - [ ] Add IT test for example 4 (game stats). Pending on on a fix for BEAM-4534.

### CODE REVIEW

**Comment by mariapython** [comment]
R: @aaltay 
cc: @angoenka (leaving the google.cloud import, like in [here](https://github.com/apache/beam/blob/master/sdks/python/apache_beam/examples/streaming_wordcount_it_test.py#L51), per our discussion).

**Comment by mariapython** [comment]
Retest this please

File: sdks/python/apache_beam/examples/leader_board_it_test.py [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+                  'on_success_matcher': all_of(state_verifier,
+                                               bq_users_verifier,
+                                               bq_teams_verifier)}
+
+    # Register cleanup before pipeline execution.
+    self.addCleanup(utils.delete_bq_table, self.project,
+                    self.dataset.name, self.OUTPUT_TABLE_USERS)
+    self.addCleanup(utils.delete_bq_table, self.project,
+                    self.dataset.name, self.OUTPUT_TABLE_TEAMS)
+
```
Comment (aaltay) [inline sdks/python/apache_beam/examples/leader_board_it_test.py]: Do you want to move `_cleanup_pubsub` here as well? That way (a) there will be a single place for all clean up code (b) you can explicitly know the clean up execution order.

**Comment by mariapython** [comment]
Run Python PostCommit

**Comment by aaltay** [comment]
Thank you @mariapython. For the future PRs could you tag me or someone else to review (e.g. R: @aaltay) please. That way I can get notifications when there is an update on your PR.

**Comment by mariapython** [comment]
I did tag you when I created the PR. You don't see the R:@altay? It is now showing as sent 20 hours ago.

**Comment by aaltay** [comment]
Thank you, I must have missed it in the notifications.

**Comment by altay** [comment]
I think you meant to tag @aaltay, not @altay.  :)

**Pull request merged**